__tests__/parse-templates.test.ts

```diff
@@ -546,36 +523,6 @@ describe('parseTemplates', function () {
     expect(templates).toEqual(expected);
   });
 
-  it('lol`hahahaha`', function () {
-    const input = 'lol`hahaha`';
-
-    const templates = parseTemplates(input, 'foo.js', {
-      templateTag: 'template',
-    });
-
-    expect(templates).toMatchInlineSnapshot(`
-      Array [
-        Object {
-          "end": Object {
-            "0": "\`",
-            "index": 10,
-          },
-          "importIdentifier": "lol",
-          "importPath": undefined,
-          "start": Object {
-            "0": "lol\`",
-            "1": "lol",
-            "groups": undefined,
-            "index": 0,
-            "input": "lol\`hahaha\`",
-          },
-          "tagName": "lol",
-          "type": "template-literal",
-        },
-      ]
-    `);
-  });
```

Comment:
This test is a duplicate of the one I commented on just above, it's testing old / legacy behavior that was unintentional (that we matched all template literals if no template literal config was provided).